config.yml

```diff
@@ -5,27 +5,51 @@ template:
   name: partner-bootcamp-actions
 before:
   - type: createIssue
-    title: Setting up your environment!
+    title: "Activity 1: Setting up your environment!"
     body: new-issue.md
     labels:
       - provision probot app
+    store:
+      EnvironmentIssueUrl: '{{ result.data.html_url }}'
+      EnvironmentIssueNumber: '{{ result.data.number }}'
   - type: createPullRequest
-    title: Creating Azure Resources
+    title: "Activity 2: Creating Azure Resources"
     body: create-azure-resources.md
     head: creating-azure-resources
+    store:
+      AzureSetupPRNumber: '{{ result.data.number }}'
+      AzureSetupPRUrl: '{{ result.data.html_url }}'
+
+  - type: createPullRequest
+    title: "Activity 3: Add a probot feature"
+    head: add-probot-feature
+    body: probot-feature-instructions.md
+
+  - type: createPullRequest
+    title: "Activity 4: Add a CI/CD workflow"
+    head: add-ci-cd-workflow
+    body: ci-cd-instructions.md
+
+  - type: createPullRequest
+    title: "Activity 5: Add a teardown workflow"
+    head: add-teardown-workflow
+    body: teardown-instructions.md
```

Comment:
@mattdavis0351 is this the right order of activities?

Reply:
@davenowell this would also be the location for the instructions:
- `responses/probot-feature-instructions.md`
- `responses/ci-cd-instructions.md`
- `responses/teardown-instructions.md`

Reply:
1. issue for creating azure/probot
2. pull for merging the create-azure workflow
3. pull for adding/merging the ci and cd workflows
4. pull for adding a probot feature
5. pull for adding a teardown workflow